CHANGELOG.rst

```diff
@@ -94,7 +107,10 @@ Improved Documentation
 
 Deprecations
 ------------
-- TBD
+- The ``synapse.lib.httpapi.Handler.user()`` and
```

Comment:
Should be:
The ``synapse.lib.httpapi.Handler.user()`` and ``synapse.lib.httpapi.Handler.getUserBody()`` methods